hackathon_site/dashboard/frontend/src/components/teamDetail/TeamInfoTable/TeamInfoTable.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { render, screen } from "testing/utils";
+import { mockTeamMultiple } from "testing/mockData";
+
+import { PageParams } from "pages/TeamDetail/TeamDetail";
+import TeamInfoTable from "components/teamDetail/TeamInfoTable/TeamInfoTable";
+import { RouteComponentProps } from "react-router-dom";
+
+const teamDetailProps = {
```

Comment:
these props go towards the TeamDetail **page**, they're not needed for your table component so you can remove them

Reply:
@YuYing-Liang are you sure? i get the following error when i remove them in the test file:
![Screen Shot 2022-06-12 at 6 42 44 PM](https://user-images.githubusercontent.com/36282235/173256511-9aaf4cb7-ee40-4b44-9783-2833ad84b0b4.png)
![Screen Shot 2022-06-12 at 6 43 07 PM](https://user-images.githubusercontent.com/36282235/173256509-0861684e-cfd3-4f1b-9ccc-bc8bb04f4df7.png)